### ISSUE TITLE
No guide/how to? 

### ISSUE THREAD

**Issue by Lyoxxo** [guardianproject/haven#417]
Hey guys
I can't figure out how to remotely connect the camera to another phone using tor.
I can't find any tutorial or guide on how to set this up.

**Comment by faithdwsn** [comment]
Yes, a how to guide or better yet set up videos are necessary for this app. 

After watching what is available on YouTube there are holes in the "how to" of this app. There is no Help section of the app either. For example, getting the Orbot to integrate remotely with Haven is easier said than done, especially since there is no Orbot app for the iPhone just the android. So if I am trying to remotely tune into the video stream of the Android that is recording in the remote location, do I download the Orbot app onto the Android recording smartphone or the OrNET version (which is all that is available for iPhones?) onto the iPhone OR both onto both phones working concurrently? Also, there are addresses required and passwords required to get Orbot to communicate and sync up, so a video or how to guide on this would be very helpful. 

Also, the other complex issue, which came up is the event logs will not send the text messages over to my iPhone even after downloading the app "Signal" SMS messenger onto the android. 

*I saw that google disabled the regular text message capacity of phones to send texts through Haven app for some reason. But based on github messages supposedly there is a way around it with the Signal app. However, when I verified the signal app, there are logs coming in for different events but no texts are being sent to the outside iPhone. It is hard to get the two phones to communicate with one another. 

Please help explain how to fix these two things and better yet. Somebody please make a how to guide and/or videos please. 

Thank you.

**Comment by faithdwsn** [comment]
After trying to figure out how to set up everything without a guide. I found this after the fact. https://guardianproject.github.io/haven/ 

It appears to be close to a how to guide online.

**Comment by lukeswitz** [comment]
A little clarification:

- There is no two-way functionality at this time. Alerts can only be sent or viewed remotely.

- Registration to multiple devices is not currently supported. SMS functionality was removed due to Google restrictions; more on that here: #364 https://github.com/guardianproject/haven/issues/364

-Orbot has bugs. Workaround: **Enter a new password and restart the server**. If you’ve started the server before downloading Orbot then this will happen until the field value is changed for the password (issue creation in progress). 

Though the github documentation does lack a proper walkthrough, changes are always welcome: simply open a pull request by editing the Readme.md file. I hope that helps ease some frustration until we can get the proper fixes pushed. Cheers all

**Comment by lukeswitz** [comment]
Digging into the code now, there's UI overlap in nearly all views. 

Ideas on changing the onboarding experience? While adding a back button three years ago, it also added confusion for some. Motion sensitivity metrics could make more sense, and the values could be set more intuitively. 

Overall I'd believe that by defining the features and explaining the benefit of each is important in user understanding & empowerment.

Perhaps in the onboarding slides, along with verification of sensitivity; clearing up confusion and possible config issues. For instance, Orbot could be included initially w/skip option, as there have been problems setting it up afterwards for some. 

Addressing those pain points should be a good start in clearing up old issues while keeping the project fresh. 

As always, all thoughts welcome & beneficial.
